### DISCUSSION TITLE
Show the next slide when you click on an anchor tag

### DISCUSSION THREAD

**Discussion by tlhatse** [nolimits4web/swiper#5891] [General]
I'm using the swiper slide in a wordpress site. I have an anchor tag in the 1st slide. When I click on it, I want the next slide(slide #2) to appear. I tried this code and it doesn't work:

<script>
    $(document).ready(function() {
        var mySwiper = $('.swiper-container').swiper();
        $('#home-slide-1 .slide-cta').click(function(e){
          e.preventDefault();
          mySwiper.swipeTo( $(this).data('slide') );
        });
    });
</script>

 I get the error: swiper() is not a function. 

Anyone knows how to fix this?

**Comment by Arhell** [comment]
may help https://github.com/nolimits4web/swiper/discussions/5839#discussioncomment-3068393
https://stackoverflow.com/questions/39168613/implement-swiper-link-to-specific-slide
